### DISCUSSION TITLE
Breaking ending parens in copilot with Clojure

### DISCUSSION THREAD

**Discussion by maersdal** [community/community#112163] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Copilot has some nice suggestions when writing, sometimes shaving of a few seconds of writing time and correctly predicting what I want to do. I write in Clojure, which is a Lisp programming language, and I use the commonly used Calva extension. 
The problems I have appear when writing a form inside a nested s-expression, where I have all the proper opening and closing parens that I want, the paredit plugin automatically forcing closing parens. But If I accept a copilot suggestion, it will sometimes destroy the terminal parens of the expression!

Possibly related/dupe: https://github.com/orgs/community/discussions/58690

It's just a bit annoying.

**Comment by DeandrePetersen** [comment]
If Copilot's suggestions are interfering with paredit's automatic insertion of parentheses, you might want to disable automatic parenthesis insertion in Copilot. This can typically be done through the settings or preferences of your code editor. Instead of relying solely on automatic parenthesis insertion, consider manually managing parentheses when accepting Copilot suggestions. After accepting a suggestion, review the surrounding parentheses and adjust them as needed to ensure that your nested s-expressions are properly formed.

**Reply by maersdal** [reply]
Ah. No such option in VS code unfortunately, or I might be blind?

**Reply by DeandrePetersen** [reply]
Oh no you are right, I missed it!
